### PR TITLE
Saas 12.5 fix rte timeout xdo

### PR DESCRIPTION
**Pull request by Xavier-Do**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: odoo/addons/base/wizard/base_language_install.py [hunk 1]
```diff
@@ -35,6 +35,8 @@ def lang_install(self):
         mods = self.env['ir.module.module'].search([('state', '=', 'installed')])
         mods.with_context(overwrite=self.overwrite)._update_translations(self.lang)
         self.state = 'done'
+        self.env.cr.execute('ANALYZE ir_translation')
```
Comment (nseinlet) [inline odoo/addons/base/wizard/base_language_install.py]: OK for me

**Comment by KangOl** [comment]
Actually, this patch needs to target saas-11.3 branch, where the unique index has been added.

**Comment by Xavier-Do** [comment]
@beledouxdenis point of view is to test that in last version and backport this later if no problem is detected

**Comment by Xavier-Do** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**